### DISCUSSION TITLE
Can I reference issues in one organisation from a project board in another?

### DISCUSSION THREAD

**Discussion by helena-smith** [community/community#22101] [Projects and Issues]
<p>I want to create a new project board in a new organisation, but the issues I want to include are in different organisations.<br>
The project board is for managing multiple teams that manage their own work individually in different places.<br>
I was looking at outside collaborators but I’m not sure this will give me what I want.<br>
Thanks.</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>I’m afraid a project board for an organization can only include issues from repositories owned by the organization, sorry! We’re always working to improve GitHub, and we consider every suggestion we receive, so perhaps you’d like to submit a feature request <a href="https://support.github.com/contact/feedback">through our official product feedback form</a> so that our product team can see exactly how you’d like GitHub to work?</p>
<p>(Outside collaborators are user accounts that are given access to a specific org repo, but don’t become members of the organization - you are right that they won’t help you access issues in different orgs.)</p>

**Comment by helena-smith** [comment]
<p>Thanks yamiacat. Sorry to hear I can’t already do that, but I’ll raise a feature request as you suggest and keep my fingers crossed for the future. We’re still planning to use a project with notes that include the urls to the issues in whichever organisation &amp; repo they’re in.</p>

**Comment by maxme** [comment]
<p>We would like that feature as well, we’re managing some projects across organisation and we have to use multiple boards, an external tool or a “project notes”, none of this solution is ideal.</p>

**Comment by maliberty** [comment]
This would be very useful for my project which has both a public and a private organization.

**Reply by maliberty** [reply]
Also some customers keep issues in their org that I would like to reference.
